data.py

```diff
@@ -1,7 +1,10 @@
 import csv
 
+def __gen_file_name(_class, sec):
```

Comment:
i thought might as well make this a function instead of writing it separately everytime and possible messing up